packages/utils/test/utils.test.ts

```diff
@@ -86,7 +86,8 @@ describe('Utils', () => {
     expect(Utils.isString({ var: 'plop' }), 'istring("") error').to.be.false;
   });
 
-  it('getCurrentTimestampInSecond()', () => {
+  // Skip because can fail because of a wrong timing
```

Comment:
???
What's the problem if we use sinon ?

Reply:
I've created another PR to fix it #85